### DISCUSSION TITLE
How do I copy the same object

### DISCUSSION THREAD

**Discussion by yzhsss** [mapstruct/mapstruct#2980] [Q&A] [answered]
<img width="644" alt="image" src="https://user-images.githubusercontent.com/20329077/185833013-76b41847-752c-4da1-bb83-a2f8b7c87372.png">

<img width="833" alt="image" src="https://user-images.githubusercontent.com/20329077/185832642-3eaff11a-3064-4b78-aa6f-09176ee718d2.png">

When the same object is copied, the properties are not copied

**Comment by filiphr** [comment, marked as answer]
Are you by any chance using Lombok? This happens when Lombok is used and the `lombok-mapstruct-binding` dependency is missing.

**Reply by yzhsss** [reply]
thanks
